products/linuxkernel.md

```diff
@@ -19,60 +19,62 @@ auto:
   git: https://github.com/torvalds/linux.git
   regex: ^v(?<major>0|[1-9]\d*)\.(?<minor>0|[1-9]\d*)(\.(?<patch>0|[1-9]\d*))?$
 releases:
-  - releaseCycle: "5.17"
+-   releaseCycle: "5.17"
     cycleShortHand: 517
-    release: "5.17"
+    release: 2022-03-20
     eol: false
-    latest: "5.17.8"
-    
-  - releaseCycle: "5.16"
+    latest: "5.17.7"
```

Comment:
This has gone back because the tags aren't on GitHub yet: https://github.com/gregkh/linux/tags